lib/src/common/model/pulls.dart

```diff
@@ -46,6 +47,10 @@ class PullRequest {
   /// Pull Request ID
   int? id;
 
+  /// Unique node identification string.
+  @JsonKey(name: 'node_id')
```

Comment:
This isn't necessary as by default it's snake_case

Reply:
Ah okay. I will remove it.